x-pack/test/apm_api_integration/tests/alerts/error_count_threshold.spec.ts

```diff
@@ -0,0 +1,181 @@
+        alertsCount: alertsCount.body.alertsCount,
+      };
+    }
+
+    before(async () => {
+      const serviceA = apm
+        .service({ name: 'service-a', environment: 'production', agentName: 'java' })
+        .instance('a');
+      const events = timerange(new Date(start).getTime(), new Date(end).getTime())
+        .ratePerMinute(1)
+        .generator((timestamp) => {
+          return serviceA
+            .transaction({ transactionName: 'tx' })
+            .timestamp(timestamp)
+            .duration(100)
+            .failure()
+            .errors(
+              serviceA
+                .error({ message: '[ResponseError] index_not_found_exception' })
+                .timestamp(timestamp + 50)
+            );
+        });
+      await synthtraceEsClient.index(events);
+    });
+
+    after(async () => {
+      await synthtraceEsClient.clean();
+      await supertest.delete(`/api/alerting/rule/${ruleId}`).set('kbn-xsrf', 'foo');
+      await supertest.delete(`/api/actions/connector/${actionId}`).set('kbn-xsrf', 'foo');
+      await es.deleteByQuery({
+        index: INDEX_NAME,
+        query: { match_all: {} },
+      });
+    });
+
+    describe('create alert', () => {
+      before(async () => {
+        actionId = await createIndexConnector({
+          supertest,
+          name: 'Error count API test',
+          indexName: INDEX_NAME,
+        });
+        const createdRule = await createAlertingRule({
+          supertest,
+          ruleTypeId: ApmRuleType.ErrorCount,
+          name: 'Apm error count',
+          params: {
+            environment: 'production',
+            threshold: 1,
+            windowSize: 99,
+            windowUnit: 'y',
+          },
+          actions: [
+            {
+              group: 'threshold_met',
+              id: actionId,
+              params: {
+                documents: [{ message: errorCountMessage }],
+              },
+              frequency: {
+                notify_when: 'onActionGroupChange',
+                throttle: null,
+                summary: false,
+              },
+            },
+          ],
+        });
+        ruleId = createdRule.id;
+      });
+
+      it('checks if alert is active', async () => {
+        const executionStatus = await waitForRuleStatus({
+          id: ruleId,
+          expectedStatus: 'active',
+          supertest,
+          log,
+        });
+        expect(executionStatus.status).to.be('active');
+      });
+
+      it('returns correct message', async () => {
+        const resp = await waitForDocumentInIndex<{ message: string }>({
+          es,
+          indexName: INDEX_NAME,
+          log,
+        });
+
+        expect(resp.hits.hits[0]._source?.message).eql(
+          `Apm error count alert is firing because of the following conditions:
+
+- Service name: service-a
+- Environment: production
+- Threshold: 1
+- Triggered value: 15 errors over the last 99y`
```

Comment:
"99y"? Should that say "99 years" ?

Reply:
giving it a second thought, and it does make sense for us to translate it though.